### PR TITLE
Conclusion changes

### PR DESCRIPTION
**Pull request by robinpiets**
Fixed:
- Fixed bug with displaying List on Print
- Separated and styled the Disclaimer
- Fixed styles for ConclusionOutcome
- Updated text: Added period
- Updated NewCheckerModal errors
- Updated print styling
- Updated texts

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/atoms/List.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ type Props = {
 };
 
 export default styled(List)<Props>`
+  -webkit-print-color-adjust: exact;
```
Comment (svenjens) [inline apps/client/src/atoms/List.ts:9]: Do we only need this for webkit browsers ?

File: apps/client/src/config/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-type Topic = {
+export type Topic = {
```
Comment (svenjens) [inline apps/client/src/config/index.ts]: Why do we export topic here? I don't see it imported anywhere

**Review by svenjens — CHANGES_REQUESTED** [review]

File: apps/client/src/components/Conclusion/NewCheckerModal.tsx [hunk 3]
```diff
@@ -87,6 +87,7 @@ const NewCheckerModal: React.FC<{
             <Radio
               checked={saveAddress === true}
```
Comment (svenjens) [inline apps/client/src/components/Conclusion/NewCheckerModal.tsx:88]: I thought we said we're going to enable yes on default?

**Review by svenjens — CHANGES_REQUESTED** [review]

File: apps/client/src/atoms/List.ts (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/atoms/List.ts:9]: I believe this is the only to force print the background colors )-:

File: apps/client/src/components/Conclusion/NewCheckerModal.tsx (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/Conclusion/NewCheckerModal.tsx:88]: No we leave it empty, but force the user to chose buy giving an error when leaving empty

File: apps/client/src/config/index.ts (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/config/index.ts]: Good point. Fixed

**Pull request merged**